docs/home/howto/CLUSTER.md

```diff
@@ -795,13 +795,14 @@ Epiphany has the ability to automaticly scale and cluster certain components on
 
 Then when applying the changed configuration using Epicli additional VM's will be spawned and configured or removed. The following components support scaling/clustering:
 
+- repository: In standard Epiphany deploment you need just one repository VM. Scaling up repository component will create a new standalone VM. Scaling down will complete remove it (LIFO - Last In, First Out). However even if you create more then one VM, by default all other components will use the first one.
```

Comment:
- repository: In standard Epiphany deployment only one repository machine is required. Scaling up repository component will create a new standalone VM. Scaling down will completely remove it (LIFO - Last In, First Out). However even if you create more then one VM, by default all other components will use the first one.